systemtest/src/main/java/io/strimzi/systemtest/resources/operator/OlmResource.java

```diff
@@ -6,71 +6,266 @@
 
 import io.strimzi.systemtest.Constants;
 import io.strimzi.systemtest.Environment;
+import io.strimzi.systemtest.enums.OlmInstallationStrategy;
 import io.strimzi.systemtest.resources.ResourceManager;
 import io.strimzi.systemtest.utils.kubeUtils.controllers.DeploymentUtils;
 import io.strimzi.test.TestUtils;
+import io.strimzi.test.executor.Exec;
 import io.strimzi.test.k8s.KubeClusterResource;
 import io.vertx.core.json.JsonArray;
 import io.vertx.core.json.JsonObject;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 
+import java.io.ByteArrayInputStream;
 import java.io.File;
 import java.io.IOException;
 import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.StandardCopyOption;
 import java.util.HashMap;
 import java.util.Map;
 import java.util.stream.Collectors;
 
 import static io.strimzi.systemtest.resources.ResourceManager.CR_CREATION_TIMEOUT;
+import static io.strimzi.test.k8s.KubeClusterResource.cmdKubeClient;
+import static io.strimzi.test.k8s.KubeClusterResource.kubeClient;
 
 public class OlmResource {
     private static final Logger LOGGER = LogManager.getLogger(OlmResource.class);
 
+    private static final String CO_POD_PREFIX_NAME = "strimzi-cluster-operator-";
```

Comment:
You can use `kubeClient().getClusterOperatorPodName();` in methods where you checking if the pod exist. If it returns the null or something, you can handle it. Just suggestion.

Reply:
But then you will need to handle the NPE ... so maybe that's not good idea :smile: . Other suggestion is to add it to `Constants`..

Reply:
Constants sounds fine!